fix-AppData-file.patch

```diff
@@ -1,13 +1,23 @@
 diff --git a/share/metainfo/torbrowser.appdata.xml b/share/metainfo/torbrowser.appdata.xml
-index 5577868..dfc51fe 100644
+index 5577868..f4b6d67 100644
 --- a/share/metainfo/torbrowser.appdata.xml
 +++ b/share/metainfo/torbrowser.appdata.xml
 @@ -1,7 +1,7 @@
  <?xml version="1.0" encoding="UTF-8"?>
  <!-- Copyright 2014 Micah Lee <micah@micahflee.com> -->
- <component>
+-<component>
 - <id>org.torproject.torbrowser.desktop</id>
++<component type="desktop">
```

Comment:
I would use a desktop-application component here.

Reply:
Oops, you're right, thanks!

Reply:
See https://www.freedesktop.org/software/appstream/docs/sect-Metadata-Application.html

>  The desktop-application component type is the same as the desktop component type - desktop is the older type identifier for desktop-applications and should not be used for new metainfo files, unless compatibility with very old AppStream tools (pre 2016) is still wanted.